### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by qwibbler**
- [ ]     Use React documentation.
- [ ]     Use React components.
- [ ]     Use React props.
- [ ]     Use React Router.
- [ ]     Connect React and Redux.
- [ ]     Handle events in a React app.
- [ ]     Write integration tests with a React testing library.
- [ ]     Use styles in a React app.
- [ ]     Use React life cycle methods.
- [ ]     Apply React best practices and language style guides in code.
- [ ]     Use store, actions and reducers in React.

Video: https://www.loom.com/share/d40f5367c7ef48048fb699073566f354

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,80 @@
...
-### Advanced Configuration
+👤 **Humaira Zaman**
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+- GitHub: [@qwibbler](https://github.com/qwibbler)
+- Twitter: [@hmemaz](https://twitter.com/hmemaz)
+- LinkedIn: [Humaira Zaman](https://www.linkedin.com/in/hmemaz1994/)
 
-### Deployment
+## 🤝 Contributing
```
Comment (omar-labana) [inline README.md:63]: - [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -1,70 +1,80 @@
...
+- React
+- Redux
+- Jest
 
-### `npm test`
+## Live Demo
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+[Live Demo Link](https://livedemo.com)
```
Comment (omar-labana) [inline README.md:19]: - [ ] Kindly, update the live demo link.

File: src/App.js [hunk 3]
```diff
@@ -1,23 +1,37 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Router>
+        <Header />
+        <Routes>
+          <Route exact path="/" element={<Home randOpacity={randOpacity} />} />
```
Comment (omar-labana) [inline src/App.js:22]: - [ ] The Home page should have list of categories that could be filtered using given parameters, e.g. country name. I'd suggest creating stateless controlled component for filtering the content based on a text query eg.  country name.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @qwibbler, 

Excellent work so far! 👌
You have implemented almost all the required features correctly and without any errors. ✔️ 
README is quite neat and tests are passing. 👏
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
- [x] I can't seem to find your video presentation. Kindly, record a video presenting your project,  
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (qwibbler) [reply README.md:63]: Thank you for the review, I'm so embarrassed, somehow my latest changes to the readme didn't get updated, I will get right on that.

File: src/App.js (hunk 3 above)
Comment (qwibbler) [reply src/App.js:22]: Can you explain what stateless controlled components are to me? I couldn't figure out what was meant with them, so currently I'm filtering by continents only, but if you could explain, I would be very grateful.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @qwibbler, 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [ ] Countries search is not showing anything. Kindly check out this [Video](https://www.loom.com/share/bc2b56a46245410e968031e9610c398c) to see what is going on.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by qwibbler** [comment]
There was a timezone error with the API. As such, I am now automatically getting up to _yesterday's_ data, not today's. This seems to have fixed it.

**Review by Lameck1 — APPROVED** [review]
Hi, @qwibbler,

Your project is complete!  It's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**